src/client/pages/Offer/AppPromotionSection.tsx

```diff
@@ -23,11 +25,12 @@ const PromotionImage = styled(LocalizedImage)`
   }
 `
 
-type Promotion = {
-  headingText: string
-  bodyText: string
-  imageUrl: string
-}
+const StyledAnchor = styled.a({
+  color: colorsV3.gray900,
+  '&:hover': {
+    color: colorsV3.gray900,
```

Comment:
Nit: Do we have any hover effect besides cursor change?  Maybe different color or opacity change will make link more obvious to user

Reply:
In this case I don't want the anchor to reveal its identity so to speak since it's just an `<a name="promotion-switch">` not an `href`